src/praisonai/praisonai/sandbox/_compat.py

```
@@ -0,0 +1,175 @@
+                    info = await self._compute.provision(config)
+                    self._instance_id = getattr(info, 'instance_id', 'default')
+                    
+                self._is_running = True
+                logger.debug(f"Started compute instance: {self._instance_id}")
+            else:
+                # Direct start method
+                await self._compute.start()
+                self._is_running = True
+                self._instance_id = "default"
+                
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)